### PR TITLE
[local-volume] Updated READMEs to include 1.8

### PR DESCRIPTION
**Pull request by ianchakeres**
Added 1.8 as alpha to READMEs, and mention 1.9 for upcoming features.

### CODE REVIEW

File: local-volume/README.md [hunk 1]
```diff
@@ -27,19 +27,19 @@ What works:
...
         * Give your pods high priority.
         * Run a workaround controller that unbinds PVCs for pods that are
           stuck pending. TODO: add link
 * External provisioner cannot correctly detect capacity of mounts added after it
   has been started.
-    * This requires mount propagation to work, which is targeted for 1.8.
+    * This requires mount propagation to work, which is targeted for 1.9.
     * Workaround: Before adding any new mount points, stop the daemonset, add
       the new mount points, start the daemonset.
 * Fsgroup conflict if multiple pods using the same PVC specify different fsgroup
```
Comment (msau42) [inline local-volume/README.md]: The fsgroup issue has been resolved in 1.8 and can be removed here

Comment (ianchakeres) [reply local-volume/README.md]: Ok. Removed.

**Pull request merged**